protomap/proto_test.go

```diff
@@ -289,3 +289,53 @@ func TestPathsFromProtoInternal(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoFromPathsInternal(t *testing.T) {
```

Comment:
Why is this called Internal?

Reply:
Fixed -- this was a hangover from a previous design.